Server/modules/IAccountManager.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
```

Comment:
Header with license information is missing

Reply:
Thats the least of my problems here. We need to make it work im the first place